packages/babel-plugin/src/css-map/__tests__/index.test.ts

```diff
@@ -36,6 +36,25 @@ describe('css map basic functionality', () => {
     ]);
   });
 
+  it('should transform css map even when the styles are defined below the component', () => {
```

Comment:
Before, this test would actually throw a recursion error from finding member expressions—which maybe my recent change introduced the recursion error actually, so it helped me debug this.